storage_controller/src/persistence.rs

```diff
@@ -670,7 +670,9 @@ impl Persistence {
                                 .or(scheduling_policy
                                     .eq(String::from(NodeSchedulingPolicy::Draining)))
                                 .or(scheduling_policy
-                                    .eq(String::from(NodeSchedulingPolicy::Filling))),
+                                    .eq(String::from(NodeSchedulingPolicy::Filling)))
+                                .or(scheduling_policy
+                                    .eq(String::from(NodeSchedulingPolicy::Deleting))),
```

Comment:
It's a bit strange that on cancellation we set the policy back to the previous one and on restart we always set it to active. Perhaps we can just always set it back to Pause (for deleting nodes) to keep things simple. Also, can you leave a comment that this is a transient state and future work will resume deletion?